packages/datepicker/src/react/dropdown-datepicker.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { useDayzed, DateObj } from 'dayzed'
+import { useDateSelectChange } from './utils'
+import { ValueOf, HTMLPropsFor } from '@pluralsight/ps-design-system-util'
+import { slides } from '../vars'
+import { Calendar } from './calendar'
+import { CalendarDates } from './calendar-dates'
+import TextInput from '@pluralsight/ps-design-system-textinput'
+import { CalendarIcon } from '@pluralsight/ps-design-system-icon'
+
+interface DropdownDatePickerProps
+  extends Omit<HTMLPropsFor<'div'>, 'onSelect'> {
+  onSelect?: (evt: React.SyntheticEvent, dateObj: DateObj) => void
+}
+
+export const DropdownDatePicker: React.FC<DropdownDatePickerProps> = ({
```

Comment:
DropdownDatePicker was a weird name for me.  Seems unfamiliar for the base case.  Maybe just DatePicker? I don't think DatePicker would collide with other named exports.

Reply:
Agree but I like to keep it named versus default.

Reply:
^ agreed.